spec/features/global_chat_spec.rb

```diff
@@ -35,17 +30,29 @@ def post_message(message)
     end
   end
 
-  it 'shows the correct timestamp' do
-    post_message(message)
-    message = RoomMessage.all.last
-    expect(page).to have_text(message.formatted_time)
-  end
+  context 'with a posted message' do
+    let(:room_message) { RoomMessage.all.last }
```

Comment:
Why do we need this extra variable? Shouldn't the last message equal the outer `let(:message)`?
And if this is to check whether the message was saved, then I would add a new test for that rather than checking it implicitly by getting the last message from the database

Reply:
The outer message is a String, but in this tests we need to call `formatted_time` on a RoomMessage object.

Reply:
Hmm, I see. Confusing names here, maybe one could name the outer one `message_text`, `message_string` or something like that?